manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -250,6 +250,8 @@ spec:
         - proxy
         - sidecar
         env:
+        - name: AMBIENT_TYPE
```

Comment:
Should this be ISTIO_META_AMBIENT_TYPE?

Reply:
istio agent has logic for grabbing all env var prefixed with `ISTIO_META_` prefix ([here](https://github.com/istio/istio/blob/master/pkg/bootstrap/config.go#L534)).  ztunnel currently doesn't have that logic (https://github.com/istio/ztunnel/pull/296) - was not sure if that was a function we want added in ztunnel

Reply:
@howardjohn thoughts on whether we need similar env var metadata collection in ztunnel?

Reply:
Might be useful eventually but there are currently no cases when we need to pass metadata